extras/controllerClient/readme.md

```diff
@@ -41,14 +41,17 @@ You can build locally or download pre-built, details:
 * `nvdaController.h`
   - A C header file containing declarations for all the provided functions.
 
-The **`extras/controllerClient/x86` directory** also contains example usage:
+The **`extras/controllerClient/Examples` directory** also contains example usage:
```

Comment:
can the Examples folder be renamed to lowerCamelCase? This is more standard in the folder structure.

```suggestion
The **`extras/controllerClient/examples` directory** also contains example usage:
```